Description: Fix makeinfo parameters for building info page
 Upstream is not interested in this patch and considers it Debian-specific.
Author: Andriy Senkovych <jolly_roger@itblog.org.ua>
Last-Update: 2011-03-01
--- a/docs/Makefile
+++ b/docs/Makefile
@@ -4,7 +4,7 @@
 
 buildbot.info: buildbot.texinfo
 	echo "@set VERSION $(VERSION)" > version.texinfo
-	makeinfo --fill-column=70 $<
+	makeinfo --no-split $<
 
 buildbot.html: buildbot.texinfo images-png
 	echo "@set VERSION $(VERSION)" > version.texinfo
